src/views/List.jsx

```diff
@@ -45,7 +46,15 @@ export function List({ data, listToken }) {
 
 			<ul>
 				{filteredData.map((item) => (
-					<ListItem key={item.id} name={item.name} />
+					<ListItem
+						key={item.id}
+						name={item.name}
+						itemId={item.id}
+						listId={listToken} //added listId
+						dateLastPurchased={item.dateLastPurchased}
+						totalPurchases={item.totalPurchases}
+						updateItem={updateItem}
```

Comment:
I don't think that we're using `listId`, are we? If not, then we can remove that.

Rather than passing `updateItem` as a prop, we can just import it and use it directly in `ListItem`. Either way works, but importing directly in the component may just make the code a little easier to read.

Reply:
We updated the code to import updateItem instead of passing it as a prop to make it easier to read. I kept listId because from the way the code works right now it is used in updateItem.